enterprise/frontend/src/embedding-sdk/components/private/data-picker/DataSelector.unit.spec.js

```diff
@@ -0,0 +1,500 @@
+import userEvent from "@testing-library/user-event";
+
+import { createMockMetadata } from "__support__/metadata";
+import { getIcon, render, renderWithProviders, screen } from "__support__/ui";
+import { delay } from "__support__/utils";
+import {
+  createMockDatabase,
+  createMockSavedQuestionsDatabase,
+  createMockTable,
+} from "metabase-types/api/mocks";
+import {
+  SAMPLE_DB_ID,
+  createSampleDatabase,
+} from "metabase-types/api/mocks/presets";
+import {
+  createMockSettingsState,
+  createMockState,
+} from "metabase-types/store/mocks";
+
+import { UnconnectedDataSelector as DataSelector } from "./DataSelector";
```

Comment:
The import is updated, it was referencing `metabase/query_builder/components/DataSelector` previously.

https://github.com/metabase/metabase/blob/7f790fe68ca323340b207d4cbbdfa2e3a81ce0cc/frontend/src/metabase/query_builder/components/DataSelector/DataSelector.unit.spec.js#L6